terraform/aws/components/lab-results/locals_env_variables.tf

```diff
@@ -0,0 +1,84 @@
+locals {
+  environment_variables = concat(var.lab-results_environment_variables,[
+    {
+      name = "NHAIS_MESH_API_HTTP_PROXY"
+      value = var.opentest_connected ? data.aws_instance.opentest_instance.private_ip : ""
+    },
+    {
+      name  = "LAB_RESULTS_OUTBOUND_SERVER_PORT"
+      value = var.lab-results_service_container_port
+    },
+    {
+      name = "LAB_RESULTS_AMQP_BROKERS"
+      value = replace(data.aws_mq_broker.lab-results_mq_broker.instances[0].endpoints[1],"amqp+ssl","amqps") # https://www.terraform.io/docs/providers/aws/r/mq_broker.html#attributes-reference
+    },
+    {
+      name = "LAB_RESULTS_OUTBOUND_QUEUE_NAME"
+      value = "${var.environment}_lab-results_gp_system_outbound"
+    },
+    {
+      name = "NHAIS_MESH_INBOUND_QUEUE_NAME"
+      value = "${var.environment}_nhais_mesh_inbound"
```

Comment:
as this and some other values will be shared between labresults and nhais it is better to have them set by variable here and in nhais, set by the same entry in etc,

Reply:
I have fixed variables in following files;

data_secrets.tf 406e05634647184d97a15b74ba8623d423dbd8b2

locals_secret_variables.tf 7ccac71e8e9408e91e252af857955621f189cf57

variables.tf b0ab685efc0ada07ef51fe30a9a8ef960ddffcd1

locals_env_variables.tf a3c89dfcbbcb3c51d38f8971b0ef794870e44466